.github/workflows/danger.yml

```diff
@@ -3,26 +3,19 @@ name: Danger
 on: [pull_request]
 
 jobs:
-  danger:
-    name: Danger JS
-    runs-on: ubuntu-latest
+  buildsh:
+    strategy:
+      matrix:
+        mode: [danger]
```

Comment:
With only one configuration, using a matrix is a bit overkill since you can just configure the job directly (like [this](https://github.com/TextureGroup/Texture/blob/master/.github/workflows/ci-master-only.yml)). Please feel free to follow up on another PR but not a big deal otherwise.